core/pdfbulkexport.class.inc.php

```diff
@@ -190,41 +190,55 @@ public function GetFooter()
 		return $sPDF;
 	}
 
+	/**
+	 * @param \DBObject $oObj
+	 * @param string $sAttCode
+	 *
+	 * @return int|string
+	 * @throws \Exception
+	 */
 	protected function GetValue($oObj, $sAttCode)
 	{
-		switch($sAttCode)
-		{
+		switch ($sAttCode) {
 			case 'id':
 				$sRet = parent::GetValue($oObj, $sAttCode);
 				break;
 
 			default:
 				$value = $oObj->Get($sAttCode);
-				if ($value instanceof ormDocument)
-				{
+				if ($value instanceof ormDocument) {
 					$oAttDef = MetaModel::GetAttributeDef(get_class($oObj), $sAttCode);
 					if ($oAttDef instanceof AttributeImage)
 					{
 						// To limit the image size in the PDF output, we have to enforce the size as height/width because max-width/max-height have no effect
 						//
 						$iDefaultMaxWidthPx = 48;
 						$iDefaultMaxHeightPx = 48;
-						if ($value->IsEmpty())
-						{
+						if ($value->IsEmpty()) {
 							$iNewWidth = $iDefaultMaxWidthPx;
 							$iNewHeight = $iDefaultMaxHeightPx;
 
 							$sUrl = $oAttDef->Get('default_image');
-						}
-						else
-						{
-							list($iWidth, $iHeight) = utils::GetImageSize($value->GetData());
+						} else {
 							$iMaxWidthPx = min($iDefaultMaxWidthPx, $oAttDef->Get('display_max_width'));
 							$iMaxHeightPx = min($iDefaultMaxHeightPx, $oAttDef->Get('display_max_height'));
 
-							$fScale = min($iMaxWidthPx / $iWidth, $iMaxHeightPx / $iHeight);
-							$iNewWidth = $iWidth * $fScale;
-							$iNewHeight = $iHeight * $fScale;
+							list($iWidth, $iHeight) = utils::GetImageSize($value->GetData());
+							if (($iWidth === 0) && ($iHeight === 0)) {
+								// avoid division by zero exception :/
+								$iNewWidth = $iDefaultMaxWidthPx;
+								$iNewHeight = $iDefaultMaxHeightPx;
+								IssueLog::Warning('AttributeImage : cannot read image size', LogChannels::EXPORT, [
```

Comment:
```suggestion
								IssueLog::Warning('AttributeImage: Cannot read image size', LogChannels::EXPORT, [
```

Shouldn't this be an error instead of a warning?

Reply:
Thanks ! About the log level, it won't be an error for svg files for example, and it shouldn't be blocking...
Kept the discussion opened so that it can be discussed on next review !